snippets/stop_RStudio.md

```diff
@@ -0,0 +1,7 @@
+> ### {% icon hands_on %} Hands-on: Stop RStudio
+>
+> If you want to stop your RStudio interactive tool, once you finish your R treatment and if relevant, saving your RStudio environment in your Galay history, you can proceed in 2 different ways:
```

Comment:
let's document here also how to save datasets, just in case. This snippet definitely needs to be used in some places :)

```suggestion
> When you have finished your R analysis, it's time to stop RStudio. 
>
> First, save your work into Galaxy, to ensure reproducibility:
> 1. You can use `gx_put(filename)` to save individual files by supplying the filename
> 2. You can use `gx_save()` to save the entire analysis transcript and any data objects loaded into your environment.
>
> Once you have saved your data, you can proceed in 2 different ways:
```